src/promptflow-devkit/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -187,21 +204,23 @@ def is_pfs_service_healthy(pfs_port) -> bool:
             return is_healthy
     except Exception:  # pylint: disable=broad-except
         pass
-    logger.debug(
-        f"Promptflow service can't be reached through port {pfs_port}, will try to (force) start promptflow service."
-    )
+    logger.debug(f"Promptflow service can't call /heartbeat on port {pfs_port}")
     return False
 
 
-def check_pfs_service_status(pfs_port, time_delay=1, count_threshold=20) -> bool:
+def check_pfs_service_status(pfs_port, time_delay=1, count_threshold=10) -> bool:
     cnt = 1
     time.sleep(time_delay)
     is_healthy = is_pfs_service_healthy(pfs_port)
     while is_healthy is False and count_threshold > cnt:
-        logger.info(
-            f"Promptflow service is not ready. It has been tried for {cnt} times, will try at most {count_threshold} "
-            f"times."
+        message = (
+            f"Promptflow service is not healthy. It has been tried for {cnt} times, will try at most "
```

Comment:
Promptflow service is not healthy -> 

Waiting for the Promptflow service status to become healthy...